src/containers/FilterNav/index.js

```diff
@@ -144,20 +143,20 @@ const FilterNavComponent = props => {
     removeCategoryFilter(category);
   };
 
-  const addPlatformFilterAction = platform => {
-    history.push(
-      location.pathname + pushToParam(rootLocationParams, 'platform', platform)
-    );
-    addPlatformFilter(platform);
-  };
-
-  const removePlatformFilterAction = platform => {
-    history.push(
-      location.pathname +
-        removeFromParam(rootLocationParams, 'platform', platform)
-    );
-    removePlatformFilter(platform);
-  };
+  // const addPlatformFilterAction = platform => {
+  //   history.push(
+  //     location.pathname + pushToParam(rootLocationParams, 'platform', platform)
+  //   );
+  //   addPlatformFilter(platform);
+  // };
+  //
+  // const removePlatformFilterAction = platform => {
+  //   history.push(
+  //     location.pathname +
+  //       removeFromParam(rootLocationParams, 'platform', platform)
+  //   );
+  //   removePlatformFilter(platform);
+  // };
```

Comment:
Why are we commenting this out?

Reply:
I don't rememberr why we are no longer using this?

Reply:
because we are using checkboxes instead of the search select, these functions were to add or remove platforms to the search select

Reply:
gotcha - can you just remove the code plz?

Reply:
done and done